fe2-android/app/src/main/java/com/github/dedis/student20_pop/home/HomeActivity.java

```diff
@@ -204,6 +260,36 @@ private void setupConnectingFragment() {
     }
   }
 
+  private void setupWalletFragment() {
+    WalletFragment walletFragment =
+        (WalletFragment) getSupportFragmentManager().findFragmentById(R.id.fragment_wallet);
+    if (walletFragment == null) {
+      walletFragment = WalletFragment.newInstance();
+      ActivityUtils.replaceFragmentInActivity(
+          getSupportFragmentManager(), walletFragment, R.id.fragment_container_home);
+    }
+  }
+
+  private void setupContentWalletFragment() {
+    ContentWalletFragment contentWalletFragment =
+        (ContentWalletFragment) getSupportFragmentManager().findFragmentById(R.id.fragment_home);
```

Comment:
Why are you using the fragment_home in this?

Reply:
The fault of copy paste. I changed it (same for setupSeedWalletFragment()).